From 204dc59286857174306615dad4b4a0bd8d0cfe82 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Caol=C3=A1n=20McNamara?= <caolanm@redhat.com>
Date: Tue, 11 Nov 2014 14:10:04 +0000
Subject: [PATCH] have a stab at improving impossible question

as to whether you want to "adapt" objects or not

Change-Id: Ic4c00be480d1be632d84dfac71fe911d48eacc51
---
 sd/source/ui/app/strings.src | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sd/source/ui/app/strings.src b/sd/source/ui/app/strings.src
index 2b363f6..06b0fcf 100644
--- a/sd/source/ui/app/strings.src
+++ b/sd/source/ui/app/strings.src
@@ -397,7 +397,7 @@ String STR_READ_DATA_ERROR
 };
 String STR_SCALE_OBJECTS
 {
-    Text [ en-US ] = "The format of the new pages will be adapted.\nDo you want to adapt the objects, too?" ;
+    Text [ en-US ] = "The page size of the target document is different than the source document.\n\nDo you want to scale the copied objects to fit the new page size?" ;
 };
 String STR_CREATE_PAGES
 {
-- 
1.9.3

